perfkitbenchmarker/linux_packages/kernel_compile.py

```diff
@@ -16,9 +16,10 @@
 
 from perfkitbenchmarker import linux_packages
 
-URL = 'https://www.kernel.org/pub/linux/kernel/v4.x/linux-4.4.25.tar.gz'
-TARBALL = 'linux-4.4.25.tar.gz'
-UNTAR_DIR = 'linux-4.4.25'
+LINUX_VERSION = '4.4.35'
+URL = 'https://www.kernel.org/pub/linux/kernel/v4.x/linux-%s.tar.gz' %(LINUX_VERSION)
+TARBALL = 'linux-%s.tar.gz' %(LINUX_VERSION)
+UNTAR_DIR = 'linux-%s' %(LINUX_VERSION)
```

Comment:
Prefer use of f-strings: https://www.python.org/dev/peps/pep-0498/

```
LINUX_VERSION = '4.4.35'
URL = f'https://www.kernel.org/pub/linux/kernel/v4.x/linux-{LINUX_VERSION}.tar.gz' 

```

Reply:
No, I have only tested this on static VMs as part of the evaluation of a single local node.

I have pushed the conversion to f-strings on top of my branch.